cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -0,0 +1,263 @@
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws","aws_sts"
+  label "AWS"
+  description "Select the AWS Credential from the list"
+  tags "provider=aws"
+end
+
+###############################################################################
+# Datasources
+###############################################################################
+
+#Generates list of Regions
+datasource "ds_regions_list" do
+  request do
+    verb "GET"
+    host "raw.githubusercontent.com"
+    path "/rightscale/policy_templates/master/data/aws/regions.json"
+    header "User-Agent", "RS Policies"
+  end
+end
+
+datasource "ds_get_caller_identity" do
+  request do
+    auth $auth_aws
+    verb "GET"
+    host "sts.amazonaws.com"
+    path "/"
+    header "User-Agent", "RS Policies"
+    query "Action", "GetCallerIdentity"
+    query "Version", "2011-06-15"
+  end
+  result do
+    encoding "xml"
+    collect xpath(response, "//GetCallerIdentityResponse/GetCallerIdentityResult") do
+      field "account",xpath(col_item, "Account")
+    end
+ end
+end
+
+datasource "ds_describe_snapshots" do
+  iterate $ds_regions_list
+  request do
+    run_script $js_describe_snapshots, val(iter_item,"region"), $ds_get_caller_identity
+  end
+  result do
+    encoding "xml"
+    collect xpath(response, "//DescribeSnapshotsResponse/snapshotSet/item", "array") do
+        field "tags" do
+          collect xpath(col_item,"tagSet/item") do
+            field "tagKey", xpath(col_item, "key")
+            field "tagValue", xpath(col_item, "value")
+        end
+      end
+        field "region",val(iter_item, "region")
+        field "snapshotId", xpath(col_item,"snapshotId")
+        field "startTime", xpath(col_item, "startTime")
+        field "ownerId", xpath(col_item, "ownerId")
+   end
+ end
+end
+
+datasource "ds_filter_old_snapshots" do
+  run_script $js_filter_old_snapshots, $snapshot_age, $ds_describe_snapshots, $param_exclude_tags
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+script "js_describe_snapshots", type: "javascript" do
+  result "results"
+  parameters "region", "ds_get_caller_identity"
+  code <<-EOS
+  results = {
+    "auth": "auth_aws",
+    "host": 'ec2.'+region+'.amazonaws.com',
+    "verb": "GET",
+    "path": "/",
+    "headers": {
+      "User-Agent": "RS Policies",
+      "Content-Type": "text/xml"
+    },
+    "query_params": {
+      'Action': 'DescribeSnapshots',
+      'Version': '2016-11-15',
+      'Filter.1.Name': 'status',
+      'Filter.1.Value.1': 'completed',
+      'Filter.2.Name': 'owner-id',
+      'Filter.2.Value.1': ds_get_caller_identity[0]['account']
```

Comment:
filtering by the owner-id using the account number as the owner should be added to the readme.

Reply:
Added